lib/barong/authorize.rb

```diff
@@ -35,6 +35,8 @@ def auth
 
     # cookies validations
     def cookie_owner
+      validate_csrf! if Barong::App.config.barong_csrf_protection
```

Comment:
I believe here we validate every single request, regardless of whether it's GET or POST etc.
However according to the official document (https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html#javascript-guidance-for-auto-inclusion-of-csrf-tokens-as-an-ajax-request-header)

POST, PUT, PATCH, DELETE and TRACE verbs, being state changing verbs, should have a CSRF token attached to the request.

Reply:
good point